qlever_dumps/props/props_text.py

```
@@ -0,0 +1,458 @@
+    # ---
+    if Old:
+        with open(claims_results_dir / "old_claims.json", "w", encoding="utf-8") as file:
+            json.dump(Old, file, indent=4)
+        # ---
+        print("Old saved to old_claims.json")
+    # ---
+    return Old
+
+
+def get_split_tab():
+    split_file = dump_files_dir / "claims_stats.json"
```

Ruff (0.13.1)

Undefined name `dump_files_dir`

(F821)